mix.exs

```diff
@@ -43,6 +45,8 @@ defmodule GenLSP.MixProject do
 
   defp docs() do
     [
+      source_url: @source_url,
```

Comment:
I think this can actually go in the project function. I checked another one of my libraries where the source code link works and it has the `source_url` there.